modules/assim.sequential/inst/anchor/IC_prep_anchorSites.Rmd

```diff
@@ -118,8 +141,8 @@ for (i in seq_along(site_info$site_id)) {
   paths <- c()
   for (j in 1:ens) {
     #unit conversion for AGB.
-    AGB <- ifelse(is.na(AGB_IC[j, i]), NULL, PEcAn.utils::ud_convert(AGB_IC[j, i], "Mg ha-1", "kg m-2"))
-    leaf_carbon_content <- ifelse(is.na(LAI_IC[j, i]), NULL, LAI_IC[j, i]/sla)#calculate leaf C based on LAI and SLA.
+    AGB <- if(is.na(AGB_IC[j, i])) NULL else {PEcAn.utils::ud_convert(AGB_IC[j, i], "Mg ha-1", "kg m-2")}
+    leaf_carbon_content <- if(is.na(LAI_IC[j, i])) NULL else {LAI_IC[j, i]/sample(sla, 1)}#calculate leaf C based on LAI and SLA.
```

Comment:
Not following this switch from `ifelse` to `if()...else`. The latter isn't going to work correctly here

Reply:
Fixed.